python/grass/temporal/list_stds.py

```diff
@@ -249,7 +265,8 @@ def _write_table(rows, column_names, output_format, separator, file):
     elif output_format == "yaml":
         _write_yaml(rows=rows, column_names=column_names, file=file)
     elif output_format == "plain":
-        # No particular reason for this separator expect that this is the original behavior.
+        # No particular reason for this separator expect that this is the original
```

Comment:
expect -> except. This is my writing actually. :facepalm:

Reply:
It has been taken care of. Feel free to let me know if you find any👍